### PR TITLE
transferring: Add maintainer metadata

### PR DESCRIPTION
**Pull request by josephholsten**

### CODE REVIEW

File: transfering-a-cookbook.md [hunk 1]
```diff
@@ -54,4 +54,12 @@ This is adapted from “[How to extract a single file with its history from a gi
 There are probably many references to the old URLs out there in the world. Some places to check
 
 - Update the `README.md` with a link to the current repo and supermarket page
-- Update the `metadata.rb` with current `issues_url` and `source_url`
+- 
+In `metadata.rb`:
+```ruby
+source_url 'https://github.com/chef-brigade/{{name}}'
+issues_url 'https://github.com/chef-brigade/{{name}}/issues'
```
Comment (zuazo) [inline transfering-a-cookbook.md]: I just want to clarify that this will break Chef 11 support :cry:

Comment (zuazo) [inline transfering-a-cookbook.md]: What do you think of the following?

``` ruby
source_url "https://github.com/chef-brigade/#{name}" if respond_to?(:source_url)
issues_url "https://github.com/chef-brigade/#{name}/issues" if respond_to?(:issues_url)
```

Comment (josephholsten) [inline transfering-a-cookbook.md]: oh no, we still want to document how to support chef 11

Also, I realized that it's more like `"https://github.com/chef-brigade/#{name}-cookbook"`

probably should standardize that. ;-)

**Comment by spheromak** [comment]
@josephholsten im :+1: on that, Was something I considered when migrating things in.

File: transfering-a-cookbook.md (hunk 1 above)
Comment (zuazo) [inline transfering-a-cookbook.md]: Go on then, this will be discussed in the skeleton anyway :+1:

**Comment by josephholsten** [comment]
@zuazo how's that?

**Comment by josephholsten** [comment]
got open PRs for each cookbook applying this

**Comment by zuazo** [comment]
Perfect! :+1:

**Pull request merged**